<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1211</width>
    <height>638</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="formLayoutWidget">
    <property name="geometry">
     <rect>
      <x>710</x>
      <y>20</y>
      <width>491</width>
      <height>461</height>
     </rect>
    </property>
    <layout class="QFormLayout" name="formLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Loaded Signals</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>Events</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QListWidget" name="listWidgetLoadFileProperty"/>
     </item>
     <item row="4" column="0" colspan="2">
      <widget class="QPushButton" name="pushButtonViewImageLoadFile">
       <property name="text">
        <string>View Signal</string>
       </property>
      </widget>
     </item>
     <item row="5" column="0" colspan="2">
      <widget class="QPushButton" name="pushButtonDeleteLoadFile">
       <property name="text">
        <string>Delete</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QListWidget" name="listWidgetLoadFile"/>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>67</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>View Signal</string>
    </property>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>50</y>
      <width>671</width>
      <height>431</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="LayoutGraph"/>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1211</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionLoadFile"/>
    <addaction name="actionExportFile"/>
    <addaction name="actionExit"/>
    <addaction name="separator"/>
   </widget>
   <widget class="QMenu" name="menuWork">
    <property name="title">
     <string>Work</string>
    </property>
    <addaction name="actionAddEventsTypes"/>
   </widget>
   <addaction name="menu"/>
   <addaction name="menuWork"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionLoadFile">
   <property name="text">
    <string>Load file</string>
   </property>
  </action>
  <action name="actionExportFile">
   <property name="text">
    <string>Export file</string>
   </property>
  </action>
  <action name="actionResizing">
   <property name="text">
    <string>Resizing</string>
   </property>
  </action>
  <action name="actionAdjustPhotoVolume">
   <property name="text">
    <string>Adjust photo volume</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionvertical">
   <property name="text">
    <string>Vertical</string>
   </property>
  </action>
  <action name="actionhorizen">
   <property name="text">
    <string>Horizen</string>
   </property>
  </action>
  <action name="actionAddEventsTypes">
   <property name="text">
    <string>Add Events Types</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
